javascript/rsd.js

```diff
@@ -0,0 +1,25 @@
+//This file is autogenerated. See modules.json and autogenerator.py for details
```

Comment:
@tybanash I see that this file is autogenerated. However, I cannot see where you have added your JSON schema in `modules.json` in this commit

Reply:
@brendajerop Oupps!!,  I will fix that right away, may be i deleted by error.